scripts/benchmarks/format-benchmark-record.js

```diff
@@ -13,14 +13,10 @@ const Constants = require('../../constants.js');
  * @enum {string}
  */
 const SUBMISSION_METHOD_MAP = {
-  'claims': 'claims',
-  'registry': 'registry',
-  'registry/qcdr': 'registry',
-  'cmswebinterface': 'cmsWebInterface',
-  'administrativeclaims': 'administrativeClaims',
-  'ehr': 'electronicHealthRecord',
-  'cmsapprovedcahpsvendor': 'certifiedSurveyVendor',
-  'cahpssurveyvendor': 'certifiedSurveyVendor'
+  'ecqm': 'electronicHealthRecord',
+  'medicarepartbclaims': 'claims',
```

Comment:
does this effect my team? We used submission method "claims"

Reply:
In the API, it will unaffected. This effectively maps from the new names back to the old submission method names which the API will continue to use.

Reply:
Perfect. thank you!